pkg/status/status.go

```diff
@@ -77,9 +77,9 @@ func GetRunStatusForPipelineTask(ctx context.Context, client versioned.Interface
 	return runStatus, nil
 }
 
-// GetFullPipelineTaskStatuses returns populated TaskRun and Run status maps for a PipelineRun from its ChildReferences.
+// GetPipelineTaskStatuses returns populated TaskRun and Run status maps for a PipelineRun from its ChildReferences.
 // If the PipelineRun has no ChildReferences, its .Status.TaskRuns and .Status.Runs will be returned instead.
-func GetFullPipelineTaskStatuses(ctx context.Context, client versioned.Interface, ns string, pr *v1beta1.PipelineRun) (map[string]*v1beta1.PipelineRunTaskRunStatus,
+func GetPipelineTaskStatuses(ctx context.Context, client versioned.Interface, ns string, pr *v1beta1.PipelineRun) (map[string]*v1beta1.PipelineRunTaskRunStatus,
```

Comment:
@abayer is this meant to be a helper function for package importers outside tektoncd/pipeline? It seems like dead code.

@JeromeJu can you please update this function and its docstring to not use taskruns/runs?

Reply:
I was not sure if this is used for populating taskruns/runs as `full` embbededStatus alike pipelineRunStatus to prepare for the removal of this feature flag? Was also wondering about this at
https://github.com/tektoncd/pipeline/issues/6090#issuecomment-1412730427

Reply:
Updated and removed the populating from taskruns/runs. Thanks for the pointers